src/components/pages/Routes.jsx

```diff
@@ -11,6 +12,7 @@ const Routes = () => {
       <Switch>
         <Route path="/" exact component={Home} />
         <Route path="/agenda" exact component={Agenda} />
+        <Route path="/real-time-monitoring" exact component={Monitoring} />
```

Comment:
```suggestion
        <Route path="/monitoring" exact component={Monitoring} />
```
juste monitoring suffit je pense, je conçois que la tu veux bien appuyer sur le fait que ca soit en temps reel